apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregningsgrunnlag/Beregningsgrunnlag.tsx

```diff
@@ -17,27 +17,55 @@ import { isPending, isSuccess } from '~shared/api/apiUtils'
 import { useFeatureEnabledMedDefault } from '~shared/hooks/useFeatureToggle'
 import styled from 'styled-components'
 import { isFailureHandler } from '~shared/api/IsFailureHandler'
+import { statusErRedigerbar } from '~components/behandling/felles/utils'
 
 const Beregningsgrunnlag = (props: { behandling: IDetaljertBehandling }) => {
   const { behandling } = props
 
-  const [overstyrtBeregning, getOverstyrtBeregning] = useApiCall(hentOverstyrBeregning)
-  const [overstyrt, setOverstyrt] = useState<OverstyrBeregning | undefined>(undefined)
+  const [overstyrtBeregningGrunnlagRest, getOverstyrBeregningGrunnlag] = useApiCall(hentOverstyrBeregningGrunnlag)
+  const [overstyrtBeregningRest, getOverstyrtBeregning] = useApiCall(hentOverstyrBeregning)
+
+  const [overstyrtBeregning, setOverstyrtBeregning] = useState<OverstyrBeregning | undefined>(undefined)
+  const [, setOverstyrtBeregningGrunnlag] = useState<OverstyrBeregningGrunnlagPostDTO | undefined>(undefined)
+
   const vedtaksresultat = useVedtaksResultat()
   const visOverstyrKnapp = useFeatureEnabledMedDefault('overstyr-beregning-knapp', false)
 
+  const [visOverstyrtBeregningGrunnlag, setVisOverstyrtBeregningGrunnlag] = useState<Boolean>(false)
+  const [erBehandlingFerdigstilt] = useState<Boolean>(!statusErRedigerbar(behandling.status))
+
   const virkningstidspunkt = behandling.virkningstidspunkt?.dato
     ? formaterDato(behandling.virkningstidspunkt.dato)
     : undefined
 
   useEffect(() => {
     getOverstyrtBeregning(behandling.id, (result) => {
       if (result) {
-        setOverstyrt(result)
+        setOverstyrtBeregning(result)
+
+        if (!erBehandlingFerdigstilt) {
+          setVisOverstyrtBeregningGrunnlag(true)
+        }
+      }
+    })
+
+    getOverstyrBeregningGrunnlag(behandling.id, (result) => {
```

Comment:
Mulig jeg leser dette feil men "if (erBehandlingFerdigstilt)" kan vel være rundt getOverstyrBeregningGrunnlag?

Reply:
Uten at jeg klarer å forstå hvorfor, så ødelegger det for visning av en ikke ferdigstilt behandling som ikke er overstyrt. Skal undersøke mer.